Remotion/Core/Core/Logging/Log4NetLog.cs

```diff
@@ -147,7 +147,7 @@ private void LogLoggingError (int eventID, Exception? exceptionObject, object me
                   new SystemStringFormat (
                       CultureInfo.InvariantCulture,
                       "Failure during logging of message:\r\n{0}\r\nEvent ID: {1}",
-                      new object[] { message.ToString(), eventID }),
+                      new object?[] { message!.ToString(), eventID }),
```

Comment:
the message-bang is a TODO bug, unless we want to fix it immediately with a null-coalesce? (not sure how we did it with similar places in the current PR. 
```suggestion
                      new object[] { message?.ToString() ?? "<no message>", eventID }),
```

Reply:
I'd rather create a separate issue.